File: PTA 1011 A+B 和 C.cpp
/* PTA 1011 A+B 和 C
 * 2022/07/25
 * */

#include<iostream>
using namespace std;

int main()
{
	int t;
	cin >> t;
	const int T = t;
	long long A[T], B[T], C[T];
	for (int i = 0; i < T; i++)
		cin >> A[i] >> B[i] >> C[i];
	for (int i = 0; i < T; i++)
	{
		if ((A[i] + B[i]) > C[i])
			cout << "Case #" << i + 1 << ": true" << endl;
		else
			cout << "Case #" << i + 1 << ": false" << endl;
	}
	return 0;
}

File: PTA 1011 A+B 和 C.java
/// PTA 1011 A+B 和 C
/// 2023/01/06

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int t = input.nextInt();
        long a, b, c;
        for (int i = 0; i < t; i++) {
            a = input.nextInt();
            b = input.nextInt();
            c = input.nextInt();
            if (a + b > c)
                System.out.printf("Case #%d: true\n", i + 1);
            else
                System.out.printf("Case #%d: false\n", i + 1);
        }
    }
}
